MAINTAINING.md

```diff
@@ -119,19 +119,31 @@ complete that no input from the original developer or maintainer is required.
 
 ### Organisation of reviewing between maintainers
 
-This section is a placeholder for further discussion around making the review
-process more efficient and effective when multiple maintainers are involved.
-This could include:
+#### Partial review
 
--   Usage of labels
--   Division of review responsibilities, e.g. with ACKs for different areas of
-    review
--   Usage of GitHub functionality, for example "Reviewers" and "Assignees" lists
+When you review a PR only partly according to the points outlined in the
+[technical guidelines](#technical-guidelines) please do not "approve" the PR to
+prevent accidental merges. Rather, give your verbal ACK and describe what you
+reviewed. In addition, if you processed or reasonably stepped over a whole
+section, mark the PR with the according label from the "Reviewed:" category.
```

Comment:
I would specifically refer to the point about skipping items. E.g.
"In addition, if you processed or reasonably stepped over a whole section, mark the PR with the according label from the "Reviewed:" category. If you step over items or sections, please articulate your reasons clearly, in accordance with the first Note in the [introduction](#riot-maintainer-guidelines) of this document."